api/inventorypb/nodes.proto

```diff
@@ -283,6 +283,16 @@ message RemoveNodeRequest {
 
 message RemoveNodeResponse {}
 
+// Check
```

Comment:
We don't need check node on pmm-admin side, it should be done on pmm-managed side while adding annotation.